controllers/etcd_controller.go

```diff
@@ -1174,8 +1168,10 @@ func (r *EtcdReconciler) getMapFromEtcd(etcd *druidv1alpha1.Etcd) (map[string]in
 		"statefulsetReplicas":     statefulsetReplicas,
 		"serviceName":             fmt.Sprintf("%s-client", etcd.Name),
 		"configMapName":           fmt.Sprintf("etcd-bootstrap-%s", string(etcd.UID[:6])),
-		"cronJobName":             getCronJobName(etcd),
 		"volumeClaimTemplateName": volumeClaimTemplateName,
+		"serviceAccountName":      fmt.Sprintf("%s-serviceaccount", etcd.Name),
+		"roleName":                fmt.Sprintf("%s-role", etcd.Name),
+		"roleBindingName":         fmt.Sprintf("%s-rolebinding", etcd.Name),
```

Comment:
Should we include something like `-br-` in the name, so that the relation to backup-store is clear?